modules/SofaGraphComponent/BackgroundSetting.h

```diff
@@ -19,18 +19,5 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#ifndef SOFA_GRAPH_COMPONENT_BACKGROUNDSETTING_H
-#define SOFA_GRAPH_COMPONENT_BACKGROUNDSETTING_H
-
-#include <SofaBaseVisual/BackgroundSetting.h>
-
-namespace sofa
-{
-namespace defaulttype
-{
-    using sofa::component::configurationsetting::BackgroundSetting ;
-} // namespace defaulttype
-} // namespace sofa
-
-
-#endif
+#pragma message "This file has been moved to a new location for release 19.06. You need to update your code by replacing #include<SofaGrapComponent/BackgroundSetting.h> by #include<SofaBaseVisual/BackgroundSetting.h"
```

Comment:
19.06 --> 19.12 :wink:

Reply:
Well.. see my last message... I'm a bit more interested to have it merged for 19.06...or remove from the ComponentChange set the following ones:
(because it was not done... and we use them)
```cpp
   // SofaOpenglVisual
    {"OglCylinderModel", Deprecated("v17.12", "v18.12")},
    {"OglGrid", Deprecated("v17.12", "v18.12")},
    {"OglRenderingSRGB", Deprecated("v17.12", "v18.12")},
    {"OglLineAxis", Deprecated("v17.12", "v18.12")},
    {"OglSceneFrame", Deprecated("v17.12", "v18.12")},
```